tests/sentry/models/test_eventuser.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from sentry.testutils import TestCase
+from sentry.models import EventUser
+
+
+class EventUserTestCase(TestCase):
+    def test_build_hash(self):
+        cases = [
+            ({'ident': 'ident', 'username': 'username', 'email': 'email', 'ip_address': '127.0.0.1'}, '67217d8b401cf5e72bbf5103d60f3e97'),
+            ({'username': 'username', 'email': 'email', 'ip_address': '127.0.0.1'}, '14c4b06b824ec593239362517f538b29'),
+            ({'email': 'email', 'ip_address': '127.0.0.1'}, '0c83f57c786a0b4a39efab23731c7ebc'),
+            ({'ip_address': '127.0.0.1'}, 'f528764d624db129b32c21fbca0cb8d6'),
+            ({}, None),
+        ]
+        for kw, value in cases:
+            assert EventUser(**kw).build_hash() == value
+
+    def test_tag_value(self):
+        cases = [
+            ({'ident': 'ident', 'username': 'username', 'email': 'email', 'ip_address': '127.0.0.1'}, 'id:ident'),
+            ({'username': 'username', 'email': 'email', 'ip_address': '127.0.0.1'}, 'username:username'),
```

Comment:
This is the case that fails on current master, since it thinks the `tag_value` should be `email:email` instead of `username:username`.